frontera/contrib/backends/redis_backend/__init__.py

```diff
@@ -331,9 +438,15 @@ def links_extracted(self, _, links):
                     )
                     links_processed.add(link_fingerprint)
                 break
+            except ConnectionError as e:
+                self._logger.exception("Connection to Redis failed when attempting to write links extracted")
+                pause = timeout.next()
+                if pause == None:
+                    break
+                sleep(pause)
             except ResponseError as e:
```

Comment:
it's definitely makes sense to wrap StrictRedis, because after you contribute this code it will be hard to remember checking all the places where some adjustments are needed in this retry/skip logic.